plugins/@grouparoo/salesforce/src/lib/export/mapping.ts

```diff
@@ -86,6 +86,7 @@ const supportedTypeMap = {
   id: "string",
   picklist: "string",
   currency: "float",
+  number: "float",
```

Comment:
is this a real type we missed before?

Reply:
yes, I saw it here: 
![image](https://user-images.githubusercontent.com/633081/144280578-bee45d14-79cb-4118-906c-473bf096601a.png)

Reply:
I think that might be "integer" in the code. Did you see it for real in the types returned? I guess it doens't hurt anyone if it's extra